test/e2e/specs/editor/various/taxonomies.spec.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+function generateRandomNumber() {
+	return Math.round( 1 + Math.random() * ( Number.MAX_SAFE_INTEGER - 1 ) );
+}
+
+test.describe( 'Taxonomies', () => {
+	test.beforeEach( async ( { admin, editor } ) => {
+		await admin.createNewPost();
+		await editor.openDocumentSettingsSidebar();
+	} );
+
+	test( 'should be able to open the categories panel and create a new main category', async ( {
+		editor,
+		page,
+	} ) => {
+		// Open the Document -> Categories panel.
+		const panelToggle = page.getByRole( 'button', {
+			name: 'Categories',
+		} );
+
+		if (
+			( await panelToggle.getAttribute( 'aria-expanded' ) ) === 'false'
+		) {
+			await panelToggle.click();
+		}
+
+		await page
+			.getByRole( 'button', {
+				name: 'Add New Category',
+				expanded: false,
```

Comment:
This line seems unnecessary.

Reply:
The panel has two `Add New Category` buttons; the first expands the form, and the second submits. I added the expanded check as a safeguard and to highlight the intention.

## Screenshot
![CleanShot 2024-01-09 at 12 08 23](https://github.com/WordPress/gutenberg/assets/240569/f070b536-965c-4d00-8b64-763c880d790f)

Reply:
Oh, I missed that 😅 If so, there is no problem.